components/form/ServicePorts.vue

```diff
@@ -195,8 +195,8 @@ export default {
             v-else
             v-model.number="row.nodePort"
             type="number"
-            min="1"
-            max="65535"
+            min="30000"
+            max="32767"
```

Comment:
The port range for node ports is configurable (per-cluster) and shouldn't be hardcoded and assumed to be the default here

Reply:
Reverted back to the previous bound check.